modules/sfm/samples/match_reconstruct.cpp

```diff
@@ -0,0 +1,106 @@
642.36511438018294, 1094.4753756517703, 638.78182172965023, 1112.2243137486491, 629.11764825424234, 1124.6020757977692, 614.32040578935505, 1130.3280000880559, 595.89479610377475, 1128.832027152311, 575.73304654345543, 1120.3140614945582, 555.89461970733043, 1105.7028037736306, 538.37099193358495, 1028.8265208422474, 630.75473327994234, 1042.8631608146595, 634.84707689081199, 1058.2584967688865, 633.36513780784253, 1073.5201808839392, 626.49673981988565, 1087.1901914406481, 614.93311570256458, 1097.9739086020063, 599.79440917844522, 1104.8505533469176, 582.52462490041466, 1107.1601342595291, 564.76447800316646, 1104.6630875159808, 548.20938486875832, 1097.5686227034093, 534.45972996765431, 1028.8265208422474, 630.75473327994234, 1030.5094451565753, 627.66651801138744, 1034.7710087258772, 619.84642439779543, 1041.1508028774458, 608.13931580035444, 1049.0276032399836, 593.68515708397672, 1057.6775534034603, 577.81224590213697, 1066.3273911603369, 561.93954098969016, 1074.203893598404, 547.48592897341746, 1080.5833098120127, 535.77951390317128, 1084.8445528041379, 527.96000855806108, 1028.8265208422474, 630.75473327994234, 1018.1741780167231, 620.74342676475555, 1011.5526455838799, 606.87287620359348, 1009.432325764166, 590.53374347926467, 1011.8840276047719, 573.23767041426277, 1018.6183086835936, 556.50494178862834, 1029.0286987776246, 541.7778391792624, 1042.2318549174643, 530.34751728162291, 1057.1072904756345, 523.28067419880733, 1072.346767864846, 521.33634501543634, 1028.8265208422474, 630.75473327994234, 1010.4101690965293, 616.63109420608566, 996.9054207091491, 599.07749404652373, 989.41722948906693, 579.80687553297571, 988.48495091991276, 560.59034617421696, 994.09748444298839, 543.12852698707104, 1005.7283601637679, 528.95005419427048, 1022.3782374179993, 519.32551513551607, 1042.6242747915535, 515.18516094478298, 1064.6864339428105, 517.0345166093, 1028.8265208422474, 630.75473327994234, 1010.1203860471132, 616.86908165559817, 996.19985447065733, 599.3243841996026, 988.33023759751575, 579.83863843873178, 987.19255503865372, 560.26867143621132, 992.84977172120693, 542.44788416870733, 1004.7580535827984, 528.03740660115034, 1021.816137753751, 518.39418344227761, 1042.44951185794, 514.45808391571939, 1064.7294016570784, 516.66192071592866, 1028.8265208422474, 630.75473327994234, 1017.5350185621438, 621.43804766425978, 1010.129091420788, 607.7697751504603, 1007.3782941632923, 591.07535887445397, 1009.5887692661379, 573.00838401730766, 1016.5588223557643, 555.38146636509168, 1027.592783283955, 539.971750817464, 1041.5742740659934, 528.32676718803498, 1057.0896910577135, 521.59772432873001, 1072.5841040247224, 520.42261009954859, 1023.6872850276745, 501.48463396235769, 1021.7363061422241, 497.10356942867941, 1149.394888427619, 563.63353085150766, 1144.1773056475884, 564.44012449568231, 975.13608212288557, 590.57245094776658, 970.39817079227953, 591.3048914910305, 1095.4493161212483, 662.63147754605677, 1093.3004875390534, 657.80612717419194));
```

Comment:
I think it's better to move the data outside of the sample. Now user can't load his own data, so sample becomes almost useless.